repack_st_arch.csh

```diff
@@ -98,8 +98,10 @@ set stages_all     = (`echo $line[3-$#line] | sed -e "s#[',]# #g"`)
 # Non-esp history output which might need to be processed.
 # "components" = generic pieces of CESM (used in the archive directory names).
 # "models" = component instance names (models, used in file names).
-set components     = (lnd  atm ice  rof)
-set models         = (clm2 cam cice mosart)
+# set components     = (lnd  atm ice  rof)
+# set models         = (clm2 cam cice mosart)
+set components     = (atm ice  rof)
```

Comment:
Why was components changed to remove lnd?

Reply:
This change is temporary, while I debugged a couple failures in this script.
I meant to change it back, but that slipped through the cracks.

Reply:
this is the kind of thing that i'm hoping that we can address by: commit the full scripts to git, modify the checked out version as needed for debugging, then 'git checkout' to restore the original version so things like this don't accumulate or get missed.  (or whatever other procedure people think will work to make these kinds of changes easier to catch.)

Reply:
It will need to be more involved than that.
Typically when I debug something I end up making 
changes that I want to keep, as well as ones that should be tossed.
I've been trying to run diffuse on every file that I'm committing,
to force me to look at all the changes to a file, but that's not
bomb proof either, apparently.  And that's one of the reasons
we have  group of people looking at all this now.  I'm as sorry 
as anyone about how tedious this all is.

Reply:
maybe you can make a copy of the version you've been tinkering with, check out a clean version, and one by one only copy over the fixes you want to keep and then do a pull request on that?  (diffuse makes it easy to select changes inside the tool.)  i believe this makes it slightly harder to copy over a mistaken change than just scanning an already changed file for differences. 

do you have 'git status' as one of the steps on your checklist?  to be sure that any fixes in scripts (except for the parameter script) are committed?

from your previous reports on how the analysis has gone wrong in the past, making changes to debug things and then forgetting to change them back seems like a recurring theme.   we might not always catch things like this because we don't know the intent of the changes.  i'm trying to help see if there is a different workflow that makes this less likely to happen.